src/profile/script.js

```diff
@@ -0,0 +1,268 @@
+  approveBtn.innerText = 'Approve';
+  rejectBtn.innerText = 'Reject';
+
+  approveBtn.onclick = async () => {
+    document.getElementById('cover-spin').style.display = 'block';
+    try {
+      const response = await fetch(`${API_BASE_URL}/users/${username}`, {
+        method: 'PATCH',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        credentials: 'include',
+        body: JSON.stringify({
+          id: profileDiffId,
+          first_name: newData.first_name || '',
+          last_name: newData.last_name || '',
+          /* email: newData.email || '',
+          phone: newData.phone || '', */
+          yoe: newData.yoe || NaN,
+          company: newData.company || '',
+          designation: newData.designation || '',
+          github_id: newData.github_id || '',
+          linkedin_id: newData.linkedin_id || '',
+          twitter_id: newData.twitter_id || '',
+          instagram_id: newData.instagram_id || '',
+          website: newData.website || '',
+        }),
+      });
+
+      if (response.ok) {
+        alert('User Data Approved !!!');
+        window.location.reload();
+      } else {
+        console.log('1');
+        alert('Something went wrong. Please check console errors.');
+      }
+    } catch (error) {
+      alert('Something went wrong. Please check console errors.');
+    } finally {
+      document.getElementById('cover-spin').style.display = 'none';
+    }
+  };
+
+  rejectBtn.onclick = async () => {
+    document.getElementById('cover-spin').style.display = 'block';
+    try {
+      const response = await fetch(`${API_BASE_URL}/users/rejectDiff`, {
+        method: 'PATCH',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        credentials: 'include',
+        body: JSON.stringify({
+          profileDiffId,
+        }),
+      });
+
+      if (response.ok) {
+        alert('User Data Rejected!!!');
+        window.location.reload();
+      } else {
+        alert('Something went wrong. Please check console errors.');
+      }
+    } catch (error) {
+      alert('Something went wrong. Please check console errors.');
+    } finally {
+      document.getElementById('cover-spin').style.display = 'none';
+    }
+  };
+
+  buttonsContainer.appendChild(approveBtn);
+  buttonsContainer.appendChild(rejectBtn);
+  document.getElementById('loader').style.display = 'none';
+}
+
+// creating a resuable card container component for showing multiple cards
+function createCardComponent({ className, tagName, innerText, parent }) {
+  const component = document.createElement(tagName);
+  if (className) {
+    component.classList.add(className);
+  }
+
+  if (innerText) {
+    component.innerText = innerText;
+  }
+
+  if (parent) {
+    parent.appendChild(component);
+  }
+
+  return component;
+}
+
+function wantedData(data) {
+  const {
+    id,
+    first_name,
+    last_name,
+    /*  email,
+    phone, */
```

Comment:
Any reasons to keep these comments?

Reply:
Done

Reply:
Earlier, we were adding those details of users too but then @lakshayman suggested not to add those details for now. Removed the comments for the time being.